common/test/acceptance/tests/lms/test_teams.py

```diff
@@ -656,6 +659,8 @@ def test_user_can_create_new_team_successfully(self):
         When I fill all the fields present with appropriate data
         And I click Create button
         Then I should see the page for my team
+        And I should see the message that says
+        "You are member of this team"
```

Comment:
Can you please indent it or move "You are member of this team" to line above it?